descope/auth/auth.go

```diff
@@ -343,15 +343,15 @@ func (auth *authenticationService) validateSession(sessionToken string, refreshT
 			logger.LogError("cannot validate refresh token, refresh expiration will not be available", tErr)
 		}
 	}
-	if sessionToken == "" || err != nil || forceRefresh {
+	if err != nil || forceRefresh {
```

Comment:
Why did you remove? If no sessionToken provided, shouldn't we refresh and return it?

Reply:
The idea is that there is no auto refresh 
I should improve the condition
Need to add the config lookup on session via cookie attribute